showtimes_gql/mutations/src/projects.rs

```diff
@@ -720,7 +729,7 @@ async fn fetch_metadata_via_anilist(
                 aired_at: Some(current_time),
             });
 
-            current_time += chrono::Duration::weeks(1);
+            current_time += jiff::SignedDuration::new(ONE_WEEK, 0);
```

Comment:
I wonder if you can use `Span::weeks(1)` here instead. That would mean using `Zoned` instead of `Timestamp`.